### DISCUSSION TITLE
How to change video player?

### DISCUSSION THREAD

**Discussion by faizalahmed804** [pystardust/ani-cli#591] [Q&A] [answered]
i know vlc and mpv are great but i want to use a different video player so can someone tell me how to do it.
i looked into the source code and tried to make changes but i ended up corrupting the program.

**Comment by CoolnsX** [comment]
Just change the player_fn variable to ur player of choice.. . Also check whether ur player supports referrer.. because it won't work without it.. also find the cli command for opening links with referrer as a argument

**Reply by faizalahmed804** [reply]
do i have to change all of these? 
![image](https://user-images.githubusercontent.com/79098407/159176421-b8d97806-bc1d-4401-924e-548ebc053a48.png)

**Comment by CoolnsX** [comment]
Just where is says player_fn="mpv"..
Also which player u want to try it with??
Bcs there is also a line u have to change(or two if u watch episodes using range)

**Reply by faizalahmed804** [reply]
I want to use parole player. For some reason I can't use vscode, vlc, mpv, discord and some other software on my computer cus whenever I launch them my computer freezes

**Comment by Derisis13** [comment, marked as answer]
In 4.0 (I think) we introduced env vars to allow this level of customization without editing the code. They are documented in the man page, the one you're looking for is `ANI_CLI_PLAYER`.
